### PR TITLE
image served as file

### PR DESCRIPTION
**Pull request by RudyMis**
PR for the image, please add comments regarding method of displaying the image

### CODE REVIEW

File: drzewa/settings.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
-
+print(BASE_DIR)
```
Comment (tonowak) [inline drzewa/settings.py]: ?

File: picture.jpg
Comment (tonowak) [inline picture.jpg:1]: Should this file be here?

File: query/views.py [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
 
 # Create your views here.
-def index(request):
+def index(_):
     return HttpResponse(b"Hello, world!")
 
 
-def location(request, loc_x, loc_y):
+def location(_, loc_x, loc_y):
+    save_loc = 'static/' + loc_x + ';' + loc_y + '.jpg'
```
Comment (tonowak) [inline query/views.py]: Looks unnecessary, I'll look into it.

File: static/map.js [hunk 3]
```diff
@@ -11,13 +11,16 @@ const tiles = L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
 
 const popup = L.popup();
 
+
 function onMapClick(e) {
+
```
Comment (tonowak) [inline static/map.js]: Any reasons for more endlines?

File: picture.jpg
Comment (RudyMis) [reply picture.jpg:1]: Removed

File: static/map.js (hunk 3 above)
Comment (RudyMis) [reply static/map.js]: No, my bad

File: drzewa/settings.py (hunk 1 above)
Comment (RudyMis) [reply drzewa/settings.py]: Removed

**Pull request merged**